runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorker.java

```diff
@@ -871,6 +871,10 @@ public void startStatusPages() {
     statusPages.start();
   }
 
+  public WorkerStatusPages getStatusPages() {
```

Comment:
Let's expose addStatusPage in the worker. Which will add the page as both servlet and capturePage.

Alternatively, we can pass `beamWorkerStatusGrpcService` to `StreamingDataflowWorker` where it can create the `sdkWorkerStatusServlet` and use it.

Reply:
done